src/components/BookForm.js

```diff
@@ -1,21 +1,69 @@
 import '../styles/BookForm.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
+// import PropTypes from 'prop-types';
```

Comment:
- [OPTIONAL] KIndly make sure to remove all commented code in order to keep the code clean and professional.